docs/patterns/components/Checkbox/index.css

```diff
@@ -0,0 +1,7 @@
+.Checkbox {
+  flex-wrap: nowrap;
+}
+
+.Checkbox .Field__label {
+  overflow-wrap: break-word;
+}
```

Comment:
This is the right approach, but we would like to make this change to the Checkbox component itself. Apologies that our ticket did not provide enough context.

Reply:
Thank you so much for your feedback. I noticed that changing `wrap` to `nowrap` breaks the layout of demo checkboxes 5 to 7. So I'm looking for another solution (since :has (which was my first thought) is not yet [supported by Firefox automatically](https://developer.mozilla.org/en-US/docs/Web/CSS/:has), I have to find another way).

Reply:
Ah, good catch. I see the broken ones you mentioned.